runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -990,6 +990,19 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 
 #endif /* defined(J9VM_GC_MODRON_SCAVENGER) */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+		if (try_scan(&scan_start, "checkpointGCthreadCount=")) {
+			UDATA checkpointGCthreadCount = 0;
+			if (!scan_udata_helper(vm, &scan_start, &checkpointGCthreadCount, "checkpointGCthreadCount=")) {
```

Comment:
there is possibility we might need to expose this to a customer, in which case this should be a 'single X' option

even long term (when we allow Xgcthread on restore side and the value is above checkpoint VM's value), we might still need to run checkpoint VM with moderate-to-high number of gc threads and reduce them at the end (rather then have 1 GC thread all the way), in which case this option might still be used

Reply:
>in which case this should be a 'single X' option
Like `-Xcheckpointgcthreads`?

Reply:
Added `-XX:CheckpointGCThreads=` and removed `-XXgc: checkpointGCthreadCount=`